src/jestTransformer.ts

```diff
@@ -20,11 +20,19 @@ function getOption(jestConfig: ProjectConfig): JestTransformerOptions {
   return {};
 }
 
-const jestTransformer: Transformer = {
+const jestTransformer: Transformer<JestTransformerOptions> = {
   getCacheKey(fileData, filename, configString) {
     return createHash(fileData + filename + configString + 'graphql-let');
   },
-  process(input, filePath, jestConfig, transformOptions) {
+  process(
+    input,
+    filePath,
+    __compatJestConfig,
```

Comment:
jest and babel-jest use arg names of `(sourceText, sourcePath, options)` for both `process` and `getCacheKey`, it would be cool to get aligned with them: https://jestjs.io/docs/next/code-transformation#writing-custom-transformers